### PR TITLE
Add auto pr workflow

### PR DESCRIPTION
**Pull request by sapiderman**
# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. Closing #744 
2. Add auto PR workflow 

## How did you implement
Created a new workflow
1. Check out the schemastore repo 
2. Get the latest schema from monika repository to update it
3. Use Github Action pete-evans/create-pull-request to create new PR to schemastore

## TO DO  
1. Please help create token "AUTO-PR-TOKEN" scoped to "repo" 

## Screenshot

It works (given the proper token)
image

### CODE REVIEW

File: .github/workflows/auto-pr-schemastore.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          wget https://raw.githubusercontent.com/hyperjumptech/monika/main/monika-config-schema.json
+
+      - name: Create Pull Request
+        id: cpr
+        uses: peter-evans/create-pull-request@v4
+        with:         
+          token: ${{ secrets.AUTO-PR-TOKEN }}
+          
+          commit-message: Updating to latest monika-config-schema 
+          committer: GitHub <noreply@github.com>
```
Comment (dennypradipta) [inline .github/workflows/auto-pr-schemastore.yml:31]: Is this GitHub by default or should we use another email instead like monika@hyperjump.tech?

Comment (sapiderman) [reply .github/workflows/auto-pr-schemastore.yml:31]: We can use any email i think. Does hyperjumptech have public ops/admin email? We can use that.

File: .github/workflows/auto-pr-schemastore.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        run: |
+          cd src/schemas/json
+          rm monika-config-schema.json
+          wget https://raw.githubusercontent.com/hyperjumptech/monika/main/monika-config-schema.json
+
+      - name: Create Pull Request
+        id: cpr
+        uses: peter-evans/create-pull-request@v4
+        with:         
+          token: ${{ secrets.AUTO-PR-TOKEN }}
```
Comment (nicnocquee) [inline .github/workflows/auto-pr-schemastore.yml:28]: secrets in Github cannot use -. I already added AUTO_PR_TOKEN secret instead.

File: .github/workflows/auto-pr-schemastore.yml (hunk 1 above)
Comment (nicnocquee) [reply .github/workflows/auto-pr-schemastore.yml:31]: why not just the same like the author `${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>`

**Review by nicnocquee — CHANGES_REQUESTED** [review]

File: .github/workflows/auto-pr-schemastore.yml (hunk 2 above)
Comment (sapiderman) [reply .github/workflows/auto-pr-schemastore.yml:28]: ok will update... Thanks!

File: .github/workflows/auto-pr-schemastore.yml (hunk 1 above)
Comment (sapiderman) [reply .github/workflows/auto-pr-schemastore.yml:31]: done.

**Review by nicnocquee — APPROVED** [review]

**Review by dennypradipta — APPROVED** [review]

**Pull request merged**